assign.py

```diff
@@ -231,17 +227,24 @@ def main():
 
         #Check to see if the workflow is a task chain or an ACDC of a taskchain
         taskchain = (schema["RequestType"] == "TaskChain") or ((schema["RequestType"] == "Resubmission") and "task" in schema["InitialTaskPath"].split("/")[1])
+
+        #Dealing with era and proc string
         if taskchain:
             # Setting the Era and ProcStr values per Task
             for key, value in schema.items():
                 if type(value) is dict and key.startswith("Task"):
```

Comment:
Hi Alli.

This line is never reached by TaskChain ACDCs. How can we iterate between the tasks of an ACDC? This line works for taskchain, not for resubmission's. E.g.:

TaskChain: https://cmsweb.cern.ch/couchdb/_utils/document.html?reqmgr_workload_cache/pdmvserv_task_HIG-RunIISpring16DR80-01109__v1_T_160608_190013_7236
Resubmission: https://cmsweb.cern.ch/couchdb/_utils/document.html?reqmgr_workload_cache/prozober_ACDC_task_HIG-RunIISpring16DR80-01109__v1_T_160613_164828_2440

Despite this, the ACDCs containing merge task are correctly assigned, as we tested on Friday. 

Regards.